ci/utils.groovy

```diff
@@ -43,6 +43,10 @@ def doGitRebase() {
     println 'Skipping rebase for canary build...'
     return
   }
+  if (params.BUILD_TYPE == 'manual') {
```

Comment:
When I say it should be opt-in, I mean reversing the logic here:
```
if (params.BUILD_TYPE == 'pr') {
   println 'Rebasing on develop since it is a PR build...'
}
```

Reply:
Yeah, I was trying to think of any other cases but I guess only PR builds need rebases. Changed.